<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zeit</string>
  </property>
  <property name="windowIcon">
   <iconset theme="chronometer">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>&amp;Zeit</string>
    </property>
    <addaction name="actionAddTask"/>
    <addaction name="actionModifyTask"/>
    <addaction name="actionDeleteTask"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPredefined">
    <property name="title">
     <string>Predefi&amp;ned</string>
    </property>
    <addaction name="actionAlarm"/>
    <addaction name="actionTimer"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuPredefined"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAddTask">
   <property name="text">
    <string>&amp;Add Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDeleteTask">
   <property name="text">
    <string>&amp;Delete Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionModifyTask">
   <property name="text">
    <string>&amp;Modify Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionTimer">
   <property name="text">
    <string>&amp;Timer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAlarm">
   <property name="text">
    <string>&amp;Alarm</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
